estate/views/estate_property/lists.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<odoo>
```

Comment:
We put all the vies of the same model inside the same file. to avoid having shit load of files when there are more than 3 models :)